bundle-workflow/src/test_workflow/perf_test_cluster.py

```diff
@@ -0,0 +1,52 @@
+import json
+import os
+import subprocess
+
+from test_workflow.test_cluster import TestCluster
+
+
+class PerformanceTestCluster(TestCluster):
+    """
+    Represents a performance test cluster. This class deploys the opensearch bundle with CDK and returns the private IP.
+    """
+
+    def __init__(self, bundle_manifest, config, stack_name, security):
+        self.manifest = bundle_manifest
+        self.security_id = config['Constants']['SecurityGroupId']
+        self.vpc_id = config['Constants']['VpcId']
+        self.account_id = config['Constants']['AccountId']
+        self.region = config['Constants']['Region']
+        self.role = config['Constants']['Role']
+        self.work_dir = 'tools/cdk/mensor/single-node/'
+        self.stack_name = stack_name
+        self.output_file = 'output.json'
+        self.ip_address = None
+        self.security = 'enable' if security else 'disable'
+        self.params = f'-c url={self.manifest.build.location} -c security_group_id={self.security_id} -c vpc_id={self.vpc_id}'\
```

Comment:
Maybe a dict that is joined with `-c`?

Reply:
Every argument is `-c`, make an array with each item on a line, and then join with "-c " as the separator. Avoids a lot of `f{}` and will be easier to diff.